### ISSUE TITLE
The requested module 'datastore-core' does not provide an export named 'Errors'

### ISSUE THREAD

**Issue by MicrowaveDev** [ipfs/js-datastore-fs#135]
Trying to execute this code:
```
import {createFactory} from 'ipfsd-ctl';
import ipfsHttpModule from 'ipfs-http-client';
import ipfsModule from 'ipfs';

const factory = createFactory({
    type: 'proc', // or 'js' to run in a separate process
    // type: 'js',
    ipfsHttpModule,
    ipfsModule, // only if you gonna spawn 'proc' controllers
});
```

Got this error:
```
file:///home/username/geesome-libs/node_modules/datastore-fs/esm/src/index.js:9
  Errors
  ^^^^^^
SyntaxError: The requested module 'datastore-core' does not provide an export named 'Errors'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:128:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:194:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:385:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)
```

Environment:
```
> node -v
v16.16.0
> lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.4 LTS
Release:        20.04
Codename:       focal
```

Package versions:
```
ipfs@0.63.5
ipfs-core@0.15.4
ipfs-core-config@0.4.1
datastore-fs@7.0.0
```

**Comment by MicrowaveDev** [comment]
Temporary fixed by rollback to 0.61.0 version of ipfs(6.0.1 version of datastore-fs).

**Comment by achingbrain** [comment]
Can you confirm which version of `datastore-core` you have?  This should have been [fixed in 7.0.3](https://github.com/ipfs/js-datastore-core/pull/111)

```console
$ npm ls datastore-core
```
